assets/js/feature-tours/unified-dashboard.js

```diff
@@ -65,9 +70,27 @@ const unifiedDashboard = {
 				'The updated Search Funnel shows you a more comprehensive view of how traffic from Search is changing.',
 				'google-site-kit'
 			),
-			placement: 'top',
+			placement: 'top-end',
```

Comment:
@asvinb, now it looks like this:
![image](https://user-images.githubusercontent.com/873049/151414005-bf7e831b-10c6-441d-8f6c-ee2becb56a5b.png)

Maybe we should better use something like the following?

```js
window.scrollTo( {
    top: tooltipElement.getBoundingClientRect().top - adminBarHeight,
    ...
} );
```

Reply:
@asvinb Hmm, wondering about that too. Right now it's all the way to the right, is that because of `top-end`? I think it would be better to have it centered above like it usually is, and I _think_ (?) `top` would do that?

Reply:
@felixarntz Actually i tried that but for whatever reason, the tooltip will be positioned to the bottom of the widget and with the scrolling logic we had implemented, the widget will be no longer visible. If we set the position to either `top-start` or `top-end`, we don't have this weird behaviour for now. 
![Jan-28-2022 13-40-08](https://user-images.githubusercontent.com/31818/151524021-f019b261-fec4-413f-b252-bb147dbcaf46.gif)

Reply:
@asvinb Thanks, it's probably okay to go with that then.